src/pages/documentation/adobe-journey-optimizer/tutorials/handle-clicks.md

```diff
@@ -75,6 +75,14 @@ The `link` parameter must be URL encoded. If you are setting this value in the A
 
 ### Open a deeplink
 
+<InlineAlert variant="help" slots="text"/>
+
+Android deeplinks require intent filters to be added in the app manifest to correctly handle deeplink URL's. Please see the Android developer documentation for more information: https://developer.android.com/training/app-links/deep-linking
+
+<InlineAlert variant="help" slots="text"/>
+
+iOS deeplinks require a custom url scheme to be setup in the application's `info` tab within Xcode to correctly handle deeplink URL's. Please see the Apple developer documenation for more information: https://developer.apple.com/documentation/xcode/defining-a-custom-url-scheme-for-your-app
```

Comment:
Same comments as above - change the `URL's` to be `URLs` and use the markdown for inserting the link.